node/node.go

```diff
@@ -223,12 +223,18 @@ func activateShhService(stack *node.Node, config *params.NodeConfig, db *leveldb
 
 		// enable mail service
 		if config.WhisperConfig.EnableMailServer {
-			if config.WhisperConfig.Password == "" {
+			if config.WhisperConfig.Password == "" && config.WhisperConfig.PasswordFile != "" {
```

Comment:
PasswordFile emptiness is already being checked inside [ReadPasswordFile](https://github.com/status-im/status-go/pull/1075/files#diff-a894bbf771e5541f902b5b16f367f99dL126), so, this is probably not needed.